src/Administration/Resources/app/administration/src/module/sw-property/page/sw-property-list/sw-property-list.spec.js

```diff
@@ -261,4 +261,32 @@ describe('module/sw-property/page/sw-property-list', () => {
 
         wrapper.vm.searchRankingService.getSearchFieldsByEntity.mockRestore();
     });
+
+    it('should not delete property group if it is in use', async () => {
+        global.activeAclRoles = ['property.deleter'];
+
+        const wrapper = await createWrapper();
+        await flushPromises();
+
+        // Mock productRepository to return results indicating usage
+        wrapper.vm.productRepository.searchIds = jest.fn(() => Promise.resolve({ data: ['some-product-id'], total: 1 }));
+
+        // Mock createNotificationError
+        wrapper.vm.createNotificationError = jest.fn();
+
+        // Mock propertyRepository.delete to ensure it's not called
+        wrapper.vm.propertyRepository.delete = jest.fn(() => Promise.resolve());
+
+        await wrapper.vm.onConfirmDelete('test-group-id');
+
+        expect(wrapper.vm.createNotificationError).toHaveBeenCalledWith({
+            message: expect.any(String), // Matches the translated error message
```

Comment:
```suggestion
            message: expect.any(String),
```
What's with all these superfluous comments?